app/assets/v2/js/cart.js

```diff
@@ -928,133 +928,207 @@ Vue.component('grants-cart', {
+        const res = await fetch(url, saveSubscriptionParams);
+        const json = await res.json();
 
-        // Gitcoin uses the zero address to represent ETH, but the contract does not. Therefore we
-        // get the value of denomination and token_address using the below logic instead of
-        // using tokenDetails.addr
-        const isEth = tokenName === 'ETH';
-        const tokenAddress = isEth ? '0x0000000000000000000000000000000000000000' : tokenDetails.addr;
-
-        // Replace undefined comments with empty strings
-        const comment = donation.grant.grant_comments === undefined ? '' : donation.grant.grant_comments;
-
-        // For automatic contributions to Gitcoin, set 'gitcoin-grant-input-amount' to 100.
-        // Why 100? Because likely no one will ever use 100% or a normal grant, so using
-        // 100 makes it easier to search the DB to find which Gitcoin donations were automatic
-        const isAutomatic = donation.grant.isAutomatic;
-        const gitcoinGrantInputAmt = isAutomatic ? 100 : Number(this.gitcoinFactorRaw);
-
-        // Add the donation parameters
-        saveSubscriptionPayload.admin_address.push(donation.grant.grant_admin_address);
-        saveSubscriptionPayload.amount_per_period.push(Number(donation.grant.grant_donation_amount));
-        saveSubscriptionPayload.comment.push(comment);
-        saveSubscriptionPayload.confirmed.push(false);
-        saveSubscriptionPayload.contract_address.push(donation.grant.grant_contract_address);
-        saveSubscriptionPayload.contract_version.push(donation.grant.grant_contract_version);
-        saveSubscriptionPayload.denomination.push(tokenAddress);
-        saveSubscriptionPayload['gitcoin-grant-input-amount'].push(gitcoinGrantInputAmt);
-        saveSubscriptionPayload.grant_id.push(grantId);
-        saveSubscriptionPayload.split_tx_id.push(txHashes[i]);
-        saveSubscriptionPayload.sub_new_approve_tx_id.push(donation.tokenApprovalTxHash);
-        saveSubscriptionPayload.token_address.push(tokenAddress);
-        saveSubscriptionPayload.token_symbol.push(tokenName);
-      } // end for each donation
-
-      // to allow , within comments
-      saveSubscriptionPayload.comment = saveSubscriptionPayload.comment.join('_,_');
-
-      // Configure request parameters
-      const url = '/grants/bulk-fund';
-      const headers = {
-        'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8'
-      };
-      const saveSubscriptionParams = {
-        method: 'POST',
-        headers,
-        body: new URLSearchParams(saveSubscriptionPayload)
-      };
+        // if (json.failures.length > 0) {
+        //   // Something went wrong, so we create a backup of the users cart
+        //   await this.manageEthereumCartJSONStore(`${userAddress} - ${new Date().getTime()}`, 'save');
+        // }
+
+        // // Clear JSON Store
+        // await this.manageEthereumCartJSONStore(userAddress, 'delete');
+      } catch (err) {
```

Comment:
is everything above here just identation changes for the try/catch? or are there logical changes?

Reply:
No logic changes in this section, just indentation